build.gradle

```diff
@@ -56,3 +57,24 @@ eclipse {
         containers 'org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8'
     }
 }
+
+jacoco {
+    toolVersion = "0.7.6.201602180812"
+    reportsDir = file("$buildDir/customJacocoReportDir")
```

Comment:
You don't have to explicitly declare `reportsDir`, this property has a default value as `"$buildDir/reports/jacoco"`

It is written on the [document](https://docs.gradle.org/current/userguide/jacoco_plugin.html).